packages/integration-tests/index.js

```diff
@@ -256,6 +256,7 @@ function setProcessEnvironment(stackName, bucketName) {
   process.env.CollectionsTable = `${stackName}-CollectionsTable`;
   process.env.ProvidersTable = `${stackName}-ProvidersTable`;
   process.env.RulesTable = `${stackName}-RulesTable`;
+  process.env.noSNS = true;
```

Comment:
Is there any reason to not go the other way and set the actual env var that will allow the publish requests to the collections topic to succeed? That would also eliminate the error message

```
process.env.collection_info_sns_topic_arn = `${stackName}-collection-information-topic`
```